packages/devtools/lib/src/settings/settings_screen.dart

```diff
@@ -0,0 +1,87 @@
+// Copyright 2019 The Chromium Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+import 'package:devtools/src/settings/settings_controller.dart';
+import 'package:vm_service/vm_service.dart';
+
+import '../framework/framework.dart';
+import '../globals.dart';
+import '../ui/analytics_platform.dart' as ga_platform;
+import '../ui/elements.dart';
+
+class FlagDetailsUI extends CoreElement {
+  FlagDetailsUI(Flag flag) : super('div', classes: 'flag-details-container') {
+    final flagDescription = div(c: 'flag-details-descriptions-container')
+      ..add(<CoreElement>[
+        CoreElement('h3', classes: 'flag-name', text: flag.name),
+        span(c: 'flag-description', text: flag.comment),
+      ]);
+
+    final flagValues = div(c: 'flag-details-values-container')
+      ..layoutVertical()
+      ..add(<CoreElement>[
+        span(c: 'flag-value', text: flag.valueAsString),
+        span(c: 'flag-modified', text: flag.modified ? 'modified' : 'default')
+      ]);
+
+    add(<CoreElement>[
+      flagDescription,
+      flagValues,
+    ]);
+  }
+}
+
+class SettingsScreen extends Screen {
+  SettingsScreen()
+      : super(
+          name: '',
+          id: 'settings',
+          iconClass: 'octicon-gear masthead-item action-button active',
+          showTab: false,
+        ) {
+    _flagList = div(c: 'flag-list-container')
+      ..layoutVertical()
+      ..add(
+        CoreElement('h2', text: 'Dart VM Flag List'),
+      );
+    _sdkVersion = div(c: 'sdk-version-container')..flex(2);
+
+    _controller = SettingsController(
+      onFlagListReady: (FlagList flagList) {
+        _flagList.add(flagList.flags.map((flag) => FlagDetailsUI(flag)));
+      },
+      onIsAnyFlutterAppReady: (bool isAnyFlutterApp) {
+        _sdkVersion
+          ..add(<CoreElement>[
+            CoreElement('h2',
+                text: '${isAnyFlutterApp ? 'Flutter' : 'Dart'} SDK Version: '),
+            span(text: serviceManager.sdkVersion),
```

Comment:
I would simplify so the text is just 
'${isAnyFlutterApp ? 'Flutter' : 'Dart'} SDK Version: ${serviceManager.sdkVersion)} '
and move this to a method in the controller as suggested in the previous comment.